homeassistant/components/vivotek/camera.py

```diff
@@ -49,10 +53,13 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
             pwd=config[CONF_PASSWORD],
         ),
         stream_source=(
-            "rtsp://%s:%s@%s:554/live.sdp",
-            config[CONF_USERNAME],
-            config[CONF_PASSWORD],
-            config[CONF_IP_ADDRESS],
+            "rtsp://%s:%s@%s:554/%s"
+            % (
+                config[CONF_USERNAME],
+                config[CONF_PASSWORD],
+                config[CONF_IP_ADDRESS],
+                config[CONF_STREAM_PATH],
+            )
```

Comment:
Why not actually use f-strings here?

Reply:
I am pretty new to Python so I was not aware of f-strings. I just read through https://realpython.com/python-f-strings/ and they look pretty cool! The article did say they were introduced in Python 3.6 so as long as we don't need this code to work in < 3.6 I am happy to make the change.

Reply:
We dropped support for python <3.6, so f-strings are fine.

Reply:
Updated